### DISCUSSION TITLE
主题颜色的搜索页面适配

### DISCUSSION THREAD

**Discussion by doio** [the1812/Bilibili-Evolved#3249] [功能建议 / Ideas]
搜索结果页面还是默认的蓝色，不知道能不能把搜索页面：
`https://search.bilibili.com/all?keyword=win` 的按钮和文本颜色也根据设置的主题颜色改变
![2022-04-16_19-10-34](https://user-images.githubusercontent.com/4461202/163672777-99e909e6-4765-4014-a3a4-e3fc3efd0fec.png)

**Comment by the1812** [comment]
返回旧版搜索页就适配了
